sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -94,11 +102,19 @@ class DependencyManager {
 
     /**
      * Triggers the project loading initially to have data in the service
+     * This is to be called after the view was initialized
      */
-    private void populateProjectService() {
-        LoadProjectsOutput output = new LoadProjectsPresenter(projectResourceService)
+    public void populateProjectService() {
+        LoadProjectsOutput output = new LoadProjectsPresenter(projectResourceService, notificationService)
         LoadProjectsInput loadProjects = new LoadProjects(loadProjectsDataSource, output)
         loadProjects.loadProjects()
```

Comment:
the service is added to the presenter and a use case instance is run to get all the projects loaded. The service is created elsewhere and only filled with data here. We need to do it this way because the subscribers have to subscribe before we fill the service. Otherwise there will be no update after subscription